helm/fiaas-skipper/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         prometheus.io/port: http5000
         prometheus.io/scrape: "true"
         checksum/config: {{ include (print $.Template.BasePath "/config_map.yaml") . | sha256sum }}
+        fiaas/log-aggregation: sumologic
```

Comment:
I think this should be a configurable value, which we override in our internal deploy (much like the values for `baseurl`, `CRD` and `TPR`).

It's not given that everyone who uses skipper is going to use sumologic for log-aggregation.

Reply:
Ok. Let me try to replace this with rendering of annotations in the template instead where we can apply those in the internal deploy.